RecoTracker/IterativeTracking/python/ExoRegionalStep_cff.py

```diff
@@ -0,0 +1,540 @@
+import FWCore.ParameterSet.Config as cms
+import RecoTracker.IterativeTracking.iterativeTkConfig as _cfg
+from Configuration.Eras.Modifier_fastSim_cff import fastSim
+
+#for dnn classifier
+from Configuration.ProcessModifiers.trackdnn_cff import trackdnn
+
+#############################################################################
+# Tracking seeded by regions of interest targeting exotic physics scenarios #
+#############################################################################
+
+exoRegionalStepClusters = _cfg.clusterRemoverForIter("ExoRegionalStep")
+for _eraName, _postfix, _era in _cfg.nonDefaultEras():
+    _era.toReplaceWith(exoRegionalStepClusters, _cfg.clusterRemoverForIter("ExoRegionalStep", _eraName, _postfix))
+
+# TRIPLET SEEDING LAYERS
+from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
+exoRegionalStepSeedLayersTripl = cms.EDProducer("SeedingLayersEDProducer",
+    layerList = cms.vstring(
+    #TOB
+    'TOB1+TOB2+MTOB3','TOB1+TOB2+MTOB4',
+    #TOB+MTEC
+    'TOB1+TOB2+MTEC1_pos','TOB1+TOB2+MTEC1_neg',
+    ),
+    TOB = cms.PSet(
+         TTRHBuilder    = cms.string('WithTrackAngle'), clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutTight')),
+         matchedRecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
+         skipClusters   = cms.InputTag('exoRegionalStepClusters')
+    ),
+    MTOB = cms.PSet(
+         TTRHBuilder    = cms.string('WithTrackAngle'), clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutTight')),
+         skipClusters   = cms.InputTag('exoRegionalStepClusters'),
+         rphiRecHits    = cms.InputTag("siStripMatchedRecHits","rphiRecHit")
+    ),
+    MTEC = cms.PSet(
+        rphiRecHits    = cms.InputTag("siStripMatchedRecHits","rphiRecHit"),
+        skipClusters = cms.InputTag('exoRegionalStepClusters'),
+        useRingSlector = cms.bool(True),
+        TTRHBuilder = cms.string('WithTrackAngle'), clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutTight')),
+        minRing = cms.int32(6),
+        maxRing = cms.int32(7)
+    )
+)
+
+# Triplet TrackingRegion
+from RecoTracker.FinalTrackSelectors.exoRegionalStepInputTracks_cfi import exoRegionalStepInputTracks
+from RecoVertex.V0Producer.generalV0Candidates_cfi import generalV0Candidates as _generalV0Candidates
+from HiggsLongLived.TreeMaker.exoRegionProducer_cfi import exoRegionProducer as _exoRegionProducer
+from HiggsLongLived.TreeMaker.trueRegionOfInterestProducer_cfi import trueRegionOfInterestProducer as _trueRegionOfInterestProducer
```

Comment:
Apparently this is breaking the tests.
this part seems inconsistent with the rest of the code; my guess is some commits or cleanup is missing.

Reply:
I missed this when moving code out of our analysis repo and obviously didn't adequately test. But it's fixed now.